templates/asset_form.html.twig

```diff
@@ -367,6 +598,141 @@
+                     'rand': rand,
+                     'min': 12,
+                     'max': 60,
+                     'step': 12,
+                     'toadd': {
+                        0: constant('Dropdown::EMPTY_VALUE'),
+                        1: _n('%d month', '%d months', 1)|format(1),
+                        2: _n('%d month', '%d months', 2)|format(2),
+                        3: _n('%d month', '%d months', 3)|format(3),
+                        6: _n('%d month', '%d months', 6)|format(6),
+                     },
+                     'unit': 'month',
+                  }) }}
+               {% endif %}
+
+               {% if item.isField('renewal') %}
+                  {{ fields.dropdownArrayField('renewal', item.fields['renewal'], {
+                     (constant('Contract::RENEWAL_NEVER')): __('Never'),
+                     (constant('Contract::RENEWAL_TACIT')): __('Tacit'),
+                     (constant('Contract::RENEWAL_EXPRESS')): __('Express'),
+                  }, __('Renewal')) }}
+               {% endif %}
+
+               {% if item.isField('max_links_allowed') %}
+                  {{ fields.numberField('max_links_allowed', item.fields['max_links_allowed'], __('Max number of items'), {
+                     'min': 1,
+                     'max': 200000,
+                     'step': 1,
+                     'toadd': {0: __('Unlimited')}
+                  }) }}
+               {% endif %}
+
+               {% if item.isField('alert') and Entity_getUsedConfig('use_contracts_alert') %}
+                  {% set alert_html %}
+                     {{ displayLastAlert('Contract', item.fields['id']) }}
+                  {% endset %}
+                  {{ fields.dropdownArrayField('alert', item.fields['alert'], item.getAlertName(), __('Email alarms'), {
+                     'add_field_html': "<span class='text-muted'>" ~ alert_html ~ "</span>"
+                  }) }}
+               {% endif %}
+
+               {% if item.getType == 'Contract' %}
+                  {# Support hours #}
+                  {{ fields.largeTitle(__('Support hours')) }}
+                  {{ fields.smallTitle(__('On week')) }}
+                  {{ fields.dropdownHoursField('week_begin_hour', item.fields['week_begin_hour'], __('Start')) }}
+                  {{ fields.dropdownHoursField('week_end_hour', item.fields['week_end_hour'], __('End')) }}
+
+                  {{ fields.smallTitle(__('On Saturday')) }}
+                  {{ fields.dropdownYesNo('use_saturday', item.fields['use_saturday'], __('Use Saturday')) }}
+                  {{ fields.dropdownHoursField('saturday_begin_hour', item.fields['saturday_begin_hour'], __('Start')) }}
+                  {{ fields.dropdownHoursField('saturday_end_hour', item.fields['saturday_end_hour'], __('End')) }}
+
+                  {{ fields.smallTitle(__('Sundays and holidays')) }}
+                  {{ fields.dropdownYesNo('use_monday', item.fields['use_monday'], __('Use Sunday')) }}
```

Comment:
I'm not sure about this label. The group label mentions Sunday and Holidays, but the old fields in the group refer to Monday. In the old form, the dropdown was unlabeled. Is it just supposed to reefer to the start of the week which would be Sunday or Monday?

Reply:
This is indeed strange, regarding the whole old table
we set opened hours for a support on 3 lines
- during week (not possible to disable)
- saturday, with a yes/no to disable 
- sunday and holidays, with also a yes/no

I have no idea why this is called use_monday
I'm note sure this is used for any hardcoded behavior, but rather just a field to save data.

I am for renaming the field to `use_sunday`

Reply:
Done. The field name `use_monday` will need updated in the datainjection plugin's hook file. I didn't see these fields used in other plugins.